src/frontend/src/components/ActionsMenu.tsx

```diff
@@ -36,22 +38,27 @@ const ActionsMenu: React.FC<ActionsMenuProps> = ({ buttons }) => {
         id="reimbursement-request-actions-dropdown"
         onClick={handleClick}
       >
-        Actions
+        {title}
       </NERButton>
       <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
-        {buttons.map((button, index) => (
-          <MenuItem
-            key={index}
-            onClick={() => {
-              handleDropdownClose();
-              button.onClick();
-            }}
-            disabled={button.disabled}
-          >
-            <ListItemIcon>{button.icon}</ListItemIcon>
-            {button.title}
-          </MenuItem>
-        ))}
+        {buttons.map((button, index) => {
+          return (
+            <>
+              {button.dividerTop ? <Divider /> : <></>}
+              <MenuItem
+                key={index}
+                onClick={() => {
+                  handleDropdownClose();
+                  button.onClick();
+                }}
+                disabled={button.disabled}
+              >
+                <ListItemIcon>{button.icon}</ListItemIcon>
+                {button.title}
+              </MenuItem>
+            </>
+          );
+        })}
```

Comment:
ok this is a bit weird, but we should do this this way to avoid a console warning
```suggestion
        {buttons.flatMap((button, index) => {
          return [
            button.dividerTop && <Divider key={`${index}-divider`} />,
            <MenuItem
              key={index}
              onClick={() => {
                handleDropdownClose();
                button.onClick();
              }}
              disabled={button.disabled}
            >
              <ListItemIcon>{button.icon}</ListItemIcon>
              {button.title}
            </MenuItem>
          ];
        })}
```